src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1353,32 +1433,54 @@ def forward(
         )
         return_dict = return_dict if return_dict is not None else self.config.return_dict
 
-        # Embed images and text queries
-        outputs = self.image_text_embedder(
-            input_ids=input_ids,
-            pixel_values=pixel_values,
-            attention_mask=attention_mask,
-            output_attentions=output_attentions,
-            output_hidden_states=output_hidden_states,
-        )
+        query_embeds = None
+        query_mask = None
+        assert (
+            input_ids is None or query_pixel_values is None
+        ), "Both input_ids and query_pixel_values cannot be passed"
```

Comment:
I think this can be a single line:
```suggestion
        assert (input_ids is None or query_pixel_values is None), "Both input_ids and query_pixel_values cannot be passed"
```

Best way to check for this is to run:
`make fixup`

This command automatically detects bugs, applies style correction and ensures that your PR passes CI tests.